plugins/ata/fu-ata-device.c

```diff
@@ -166,6 +166,136 @@ fu_ata_device_parse_id_maybe_dell (FuAtaDevice *self, const guint16 *buf)
 	guid_efi = fu_ata_device_get_guid_safe (buf, 129);
 	if (guid_efi != NULL)
 		fu_device_add_guid (FU_DEVICE (self), guid_efi);
+
+	/* owned by Dell */
+	fu_device_set_vendor (FU_DEVICE (self), "Dell");
+	fu_device_set_vendor_id (FU_DEVICE (self), "ATA:0x1028");
+}
```

Comment:
Does this need to call fu_device_set_name still since the call moved around?

Reply:
A good question, but I don't think so. The the name should be set by FuUdevDevice already.

Reply:
OK.